src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentService.scala

```diff
@@ -117,15 +117,10 @@ class ConsignmentService(
   }
 
   def updateSeriesIdOfConsignment(updateConsignmentSeriesIdInput: UpdateConsignmentSeriesIdInput): Future[Int] = {
-    addSeriesStatus(updateConsignmentSeriesIdInput.consignmentId)
+    consignmentStatusRepository.updateConsignmentStatus(updateConsignmentSeriesIdInput.consignmentId, "Series", Completed, Timestamp.from(timeSource.now))
```

Comment:
Should probably not be updating the status until the series id has been added to the consignment. Would change this method to something like this:
  ```
def updateSeriesIdOfConsignment(updateConsignmentSeriesIdInput: UpdateConsignmentSeriesIdInput): Future[Int] = {
    for {
      result <- consignmentRepository.updateSeriesIdOfConsignment(updateConsignmentSeriesIdInput)
      seriesStatus = if (result == 1) { Completed } else { Failed }
      _ <- consignmentStatusRepository.updateConsignmentStatus(updateConsignmentSeriesIdInput.consignmentId, "Series", seriesStatus, Timestamp.from(timeSource.now))

    } yield result
  }
```

Reply:
This makes sense. I've made the changes